tests/models/anthropic/test_output.py

```diff
@@ -0,0 +1,705 @@
+"""Tests for Anthropic native JSON schema output and strict tool support.
+
+This module tests the implementation of Anthropic's structured outputs feature,
+including native JSON schema output for final responses and strict tool calling.
+
+Test organization:
+1. Strict Tools - Model Support
+2. Strict Tools - Schema Compatibility
+3. Native Output - Model Support
+4. Auto Mode Selection
+5. Beta Header Management
+6. Comprehensive Parametrized Tests - All Combinations (24 test cases)
+"""
+
+from __future__ import annotations as _annotations
+
+from collections.abc import Callable
+from typing import Annotated
+
+import httpx
+import pytest
+from inline_snapshot import snapshot
+from pydantic import BaseModel, Field
+
+from pydantic_ai import Agent
+from pydantic_ai.exceptions import UserError
+from pydantic_ai.output import NativeOutput
+
+from ...conftest import try_import
+from ..test_anthropic import MockAnthropic, get_mock_chat_completion_kwargs
+
+with try_import() as imports_successful:
+    from anthropic import AsyncAnthropic, omit as OMIT
+    from anthropic.types.beta import BetaMessage, BetaTextBlock, BetaUsage
+
+    from pydantic_ai.models.anthropic import AnthropicModel, AnthropicModelSettings
+    from pydantic_ai.providers.anthropic import AnthropicProvider
+
+from ..test_anthropic import completion_message
+
+pytestmark = [
+    pytest.mark.skipif(not imports_successful(), reason='anthropic not installed'),
+    pytest.mark.anyio,
+    pytest.mark.vcr,
+]
+
+
+# =============================================================================
+# STRICT TOOLS - Model Support
+# =============================================================================
+
+
+def test_strict_tools_supported_model_auto_enabled(
+    allow_model_requests: None, weather_tool_responses: list[BetaMessage]
+):
+    """sonnet-4-5: strict=None + compatible schema → auto strict=True + beta header."""
```

Comment:
The test name and docstring describe the behavior we actually want, but the test itself verifies the opposite 😬